dashboard/src/components/AppView/ChartInfo/ChartInfo.v2.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import { IRelease } from "shared/types";
+import "./ChartInfo.v2.css";
+import ChartUpdateInfo from "./ChartUpdateInfo";
+
+interface IChartInfoProps {
+  app: IRelease;
+}
+
+function ChartInfo({ app }: IChartInfoProps) {
+  const metadata = app.chart && app.chart.metadata;
+  let versionsInfo;
+  let descriptionInfo;
+  if (metadata) {
+    versionsInfo = (
+      <section className="chartinfo-subsection" aria-labelledby="chartinfo-versions">
+        <h5 className="chartinfo-subsection-title" id="chartinfo-versions">
+          Versions
+        </h5>
+        <div>
+          {metadata.appVersion && <div>App Version: {metadata.appVersion}</div>}
+          <span>Chart Version: {metadata.version}</span>
```

Comment:
I'd emphasize the final data here. Since both label and value are using the same style, it's not clear to me what's important in this section. Maybe showing the versions using a bold text is enough:

```suggestion
          {metadata.appVersion && <div>App Version: <strong>{metadata.appVersion}</strong></div>}
          <span>Chart Version: <strong>{metadata.version}</strong></span>
```